connect/runtime/src/main/java/org/apache/kafka/connect/runtime/isolation/DelegatingClassLoader.java

```diff
@@ -408,4 +407,18 @@ protected void scan(URL url) {
             }
         }
     }
+
+    @Override
+    public URL getResource(String name) {
+        // Default implementation of getResource searches the parent class loader. This will enable the PluginClassLoader to limit its
+        // resource search only to the URL path.
+        return null;
+    }
+
+    @Override
+    public Enumeration<URL> getResources(String name) throws IOException {
+        // Default implementation of getResources searches the parent class loader and also the URL Paths. This will enable the
```

Comment:
nit: `URL Paths` -> `URL paths` as elsewhere in the text. 
also you may say: `and also the URL Paths` -> `and also its own URL paths` to be more obvious whose URL paths we are talking about here.